packages/cactus-common/src/main/typescript/objects.ts

```diff
@@ -29,7 +29,7 @@ export class Objects {
    *
    * @param anObject
    */
-  public static getAllMethodNames(anObject: any): string[] {
+  public static getAllMethodNames(anObject: Record<string, unknown>): string[] {
```

Comment:
@akankshadixit Sorry for just noticing this, but specifically this change must be moved to a separate, brand new pull request because it is a breaking change that we cannot make prior to releasing version 2.0 (we can only make breaking changes while issuing so called major releases according to the rules of semantic versioning).

So what I ask here is for you to:
1. Open a new pull request that makes only this change (and the update to the test case that is needed to make it compile)
2. Remove this change from this particular pull request so that it contains everything else but not this change.
3. Then we can merge this pull request and will postpone the merging of the other one with the breaking change to the time when we are releasing 2.0

Sorry again for not catching this sooner.

Reply:
@akankshadixit I figured out a way to make the change needed but without having it become a breaking change! Resolving.